lib/app/file.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
-var ArrayStore = require('fh-wfm-mediator/lib/array-store');
-
-var files = [];
+var SimpleStore = require('fh-wfm-simple-store');
 
 module.exports = function(mediator) {
-  var arrayStore = new ArrayStore('file', files);
-  arrayStore.listen('cloud:', mediator);
+  var fileStore = new SimpleStore('file');
+
+  fileStore.list().then(function(res) {
+    if (!res || res.length === 0) {
```

Comment:
Ah, now I see you're checking for existing data before initializing with the seed data, nvm!